path-traversal/HADJADJDAOUD_under_dev/js.js

```
@@ -0,0 +1,617 @@
+
+      return resultText || `No relevant results found for "${query}".`;
+    } catch (error) {
+      console.error(`Web search error for "${query}":`, error.message);
+      return `Error performing web search: ${error.message}`;
+    }
+  },
+
+  calculator: async (expression) => {
+    console.log(`Calculating: "${expression}"`);
+    try {
+      const result = eval(expression);
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)